api/opencl_runtime_layer.asciidoc

```diff
@@ -16589,6 +16589,39 @@ device occupancy high by avoiding blocking in host code.
 
 [open,refpage='clUpdateMutableCommandsKHR',desc='Modify configuration of mutable-command handles to update behavior for future enqueues',type='protos']
 --
+
+==== Update to Command-Buffer Lifecycle
+
+If this extention is supported, the command-buffer lifecycle is extended to
```

Comment:
I would prefer to have a single command-buffer lifecycle diagram, defined in a way that the immutable command-buffer can do an immediate transition to an executable state, rather than maintain two separate lifecycle diagrams.